web/client/translations/data.en-US.json

```
@@ -1125,12 +1125,13 @@
             "changePwd": "Change Password",
             "newPwd": "New Password",
             "retypePwd": "Retype Password",
-            "passwordMinlenght": "Your password must be at least {minSize} character",
+            "passwordMinlenght": "Your password must be at least {data} character",
             "passwordCheckFail": "Passwords do not match!",
             "passwordInvalid": "Invalid password",
             "username": "Username",
             "password": "Password",
-            "passwordMessage": "Password must contain at least 6 characters",
+            "passwordMessage": "Password must contain at least 6 characters. Allowed characters !,@,#,$,%,&,*,_",
+            "passwordInvalidChar": "Password must not contain these characters {data}",
```

```suggestion
            "passwordInvalidChar": "The password must not contain these characters: {data}",
```
(and other translation files accordingly)

---

Made the changes.